src/interactivity/win32/windowproc.cpp

```diff
@@ -887,11 +967,6 @@ BOOL Window::PostUpdateWindowSize() const
     auto& gci = ServiceLocator::LocateGlobals().getConsoleInformation();
     const auto& ScreenInfo = GetScreenInfo();
 
-    if (ScreenInfo.ConvScreenInfo != nullptr)
-    {
-        return FALSE;

```

Comment:
Honestly, I don't know. I can't think of a single reason why the existence of an ongoing composition should prevent the window from being resized.

I suspect this may have been a limitation that affected the previous implementation with its per-line text buffers it allocated. The new implementation definitely doesn't have any problems with resizing while a composition is active.

Reply:
amazing. thanks